src/supporting-data/shared/components/EntryDownload.tsx

```diff
@@ -0,0 +1,62 @@
+import { useRouteMatch } from 'react-router-dom';
+import { DropdownButton, DownloadIcon } from 'franklin-sites';
+
+import apiUrls from '../../../shared/config/apiUrls';
+import { allSupportingDataEntryLocations } from '../../../app/config/urls';
+
+import { fileFormatEntryDownload as citationsFFED } from '../../citations/config/download';
+import { fileFormatEntryDownload as databaseFFED } from '../../database/config/download';
+import { fileFormatEntryDownload as diseasesFFED } from '../../diseases/config/download';
+import { fileFormatEntryDownload as keywordsFFED } from '../../keywords/config/download';
+import { fileFormatEntryDownload as locationsFFED } from '../../locations/config/download';
+import { fileFormatEntryDownload as taxonomyFFED } from '../../taxonomy/config/download';
+
+import { Namespace } from '../../../shared/types/namespaces';
+
+const lut = new Map([
```

Comment:
Not sure what `lut` stands for?

Reply:
Look up table. I already changed it in the following branch about the map to buttons

Reply:
Called `formatMap` in the other branch

Reply:
I am familiar with LUTs in video editing but didn't think this was about adding a vintage Kodak look to the website 😝